mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -149,7 +153,7 @@
 						"header": [
 							{
 								"key": "X-Okapi-Module-Id",
-								"value": "{{kb-ebsco-java-module-id}}",
+								"value": "mod-kb-ebsco-java-2.0.4-SNAPSHOT",
```

Comment:
it would be good not to hardcode version of the module but use the environment variable `{{kb-ebsco-java-module-id}}` to retrieve the version installed